tests/vmi_lifecycle_test.go

```diff
@@ -297,6 +297,8 @@ var _ = Describe("VMIlifecycle", func() {
 
 		Context("when virt-handler crashes", func() {
 			It("should recover and continue management", func() {
+				tests.SkipIfNoVersion(virtClient)
```

Comment:
Can we make something like `tests.SkipIfVersionBelow("1.10.4", "Device plugins have issue X in these versions.")`? Otherwise we will for sure forget why we skipped which tests and for which version.